packages/ra-auth-cognito/Readme.md

```diff
@@ -79,6 +79,7 @@ const App = () => {
            authProvider={authProvider}
            dataProvider={dataProvider}
            title="Example Admin"
+           loginPage={false} // We don't need the login page in this case
```

Comment:
I believe this is actually buggy in RA. Until the bug is fixed, we should not encourage to use that option I believe.

Reply:
As discussed, it's not

Reply:
👋🏼 I posted some questions in the Discord the other week and wanted to chime in here if that's alright.

I was chasing an issue where on logout, with a Cognito app integration that only has Google as the only identity provider, the username/password form would flash before the redirection to the AWS Hosted UI. Changing this value to `false` fixed that issue. I hadn't tried that before seeing this PR.

For reference, I'm using `react-admin` 4.7.1